packages/react-router/tests/link.test.tsx

```diff
@@ -34,6 +36,72 @@ afterEach(() => {
 })
 
 describe('Link', () => {
+
+  test('when using renderHook it returns a hook with same content to prove rerender works', async () => {
```

Comment:
Also, please use prettier 😅. Our prettier configuration is available at the root.

Reply:
I will change that :)